src/index.html

```diff
@@ -5,9 +5,55 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
i don't think `stars--0` modifier is needed

Reply:
okay noted.... let me remove it

Reply:
It seems the test requires it to search for the first block.

Reply:
yeah, damn tests